README.md

```diff
@@ -70,14 +70,15 @@ aws_secret_access_key =
 That will enable you to use the `aws` cli using `--profile local`, which means you guarantee you are not authenticating against any actual AWS account.
 
 Add the queue:
-```
-aws sqs create-queue --region us-east-1 --queue-name proxy-request-service --endpoint http://localhost:4576 --profile local
+
+```sh
+aws sqs create-queue --region us-east-1 --queue-name proxy-request-service --endpoint http://localhost:4566 --profile local
 ```
 
 When populating an SQS queue, the `aws sqs` cli tool may be useful for inspecting the messages written. For example, when populating a localstack SQS, run the following to pop the last 10 messages:
 
-```
-aws sqs receive-message --region us-east-1 --queue-url http://localhost:4576/queue/proxy-request-service --endpoint http://localhost:4576 --profile local --attribute-names All --message-attribute-names All --max-number-of-messages 10
+```sh
+aws sqs receive-message --region us-east-1 --queue-url http://localhost:4566/000000000000/proxy-request-service --endpoint http://localhost:4566 --profile local --attribute-names All --message-attribute-names All --max-number-of-messages 10
```

Comment:
Is that `000000000000` intentional?

Reply:
Yes, that's the new url structure from localstack